pages/Reference/glossary.md

```diff
@@ -4,4 +4,24 @@ route: /Reference/Glossary
 menu: Reference
 ---
 
-TBD
+# Glossary
+
+* **Account** - See "Social Identity"
+* **Announcement** - A item of data, typically an image or note, either directly posted or hashed then posted to the blockchain
```

Comment:
"A item" => "An item"

Reply:
Also, "either directly posted or hashed then posted to the blockchain" might not be accurate anymore. Maybe we should just replace that bit with "posted to the blockchain" without any details?